app/main.py

```diff
@@ -14,4 +14,10 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count_letter = 0
+
+    for i in phrase:
+        if i.lower() == letter.lower():
+            count_letter += 1
+
+    return count_letter
```

Comment:
The solution is ok, but can you try to rewrite it using just 1 line?
Also, use more informative names for variables - not "i"